functions/__sf_section_ruby.fish

```diff
@@ -29,6 +29,8 @@ function __sf_section_ruby -d "Show current version of Ruby"
 		set ruby_version (rvm-prompt i v g)
 	else if type -q rbenv
 		set ruby_version (rbenv version-name)
+  else if type -q chruby
+    set ruby_version (echo $RUBY_AUTO_VERSION)
```

Comment:
Looks like the indentation for these added lines is off, we use tabs, equivalent to 4 spaces I believe.

Reply:
Oh man I totally missed this! My bad, I'll swap these to tabs

Reply:
I believe the same thing can be achieved without the need for the nested echo command if you use the variable directly:
`set ruby_version $RUBY_AUTO_VERSION`

Reply:
Haha yeah just noticed that. Removed in the latest commit